### DISCUSSION TITLE
Wrong PDF after mogrify and convert

### DISCUSSION THREAD

**Discussion by touristB** [ImageMagick/ImageMagick#2116] [General] [answered]
Hello,
i want to build a PDF from some screenshots (size: 1080x1920) so I cut all the screenshots (.png) in the directory:
`mogrify -crop 1000x1350+20+150 *.png`
and build a PDF from the cropped files:
`convert *.png 52shots.pdf`
The cropping works nicely but the PDF looks bad. In the PDF the pages still have the size 1080x1920 so my screenshots are surrounded by a lot of white space with the cropped screenshot in the middle. 
How can I generate a nice PDF with no white borders? Any help is welcome.

bye
touristB

PS: If this matters: Version: ImageMagick 6.9.7-4 Q16 x86_64 20170114 

[52shots.pdf](https://github.com/ImageMagick/ImageMagick/files/4742095/52shots.pdf)
![Screenshot](https://user-images.githubusercontent.com/66557228/83969210-fd064f80-a8ce-11ea-8fe9-ea274de0c645.png)
![Screenshot_cropped](https://user-images.githubusercontent.com/66557228/83969213-00014000-a8cf-11ea-87b5-5badae7b8ef4.png)

**Comment by snibgo** [comment, marked as answer]
I suggest you insert "+repage" after the crop.

**Comment by touristB** [comment]
thanx snibgo!!! `+repage` did the trick :-)

So, the solution for my problem is:

`mogrify -crop 1000x1350+20+145 +repage -path cropped *.png`
